### PR TITLE
PP-18: Add nested routes for all pages

### PR DESCRIPTION
**Pull request by sanjay-techuz**

### CODE REVIEW

**Comment by sanjay-techuz** [comment]
[Add nested routes for all pages](https://trello.com/c/r36TzCLz/18-add-nested-routes-for-all-pages)

File: src/components/AppBar/index.jsx [hunk 1]
```diff
@@ -94,7 +94,7 @@ export default function MiniDrawer({ children }) {
     () => [
       {
         name: 'Project list',
-        path: '/dashboard'
+        path: '/project-list'
```
Comment (ketanp-techuz) [inline src/components/AppBar/index.jsx:97]: Instead of project-list root path will ne projects

File: src/App.js [hunk 2]
```diff
@@ -3,112 +3,58 @@ import { Routes, Route } from 'react-router-dom';
...
           element={
             <ProtectedRoute>
-              <Dashboard />
+              <ProjectList />
             </ProtectedRoute>
           }
         />
         <Route
-          path="/employee-list"
+          path="/employee-list/*"
```
Comment (ketanp-techuz) [inline src/App.js:27]: Instead of employee-list root path will ne employees

File: src/components/Table/Employee/List/ActionTableCell.jsx [hunk 3]
```diff
@@ -8,7 +8,7 @@ const ActionTableCell = (props) => {
   return (
     <Box sx={{ display: 'flex', flexDirection: 'row', justifyContent: 'center' }}>
       <ButtonGroup variant="text" aria-label="text button group">
-        <Button onClick={() => navigate(`/employee-detail/${row.id}`)}>Edit</Button>
+        <Button onClick={() => navigate(`/employee-list/detail/${row.id}`)}>Edit</Button>
```
Comment (ketanp-techuz) [inline src/components/Table/Employee/List/ActionTableCell.jsx:11]: we're using the same route for details and edit page?

Comment (sanjay-techuz) [reply src/components/Table/Employee/List/ActionTableCell.jsx:11]: for now. yes

**Review by ketanp-techuz — APPROVED** [review]

**Pull request merged**